### PR TITLE
soap client describe() resolves namespace cross references between schema

### PR DESCRIPTION
**Pull request by PaoloSarti**
Hi,

this PR adds support to cross references in namespaces between types from different schema.

### CODE REVIEW

File: lib/wsdl.js [hunk 1]
```diff
@@ -116,6 +117,12 @@ var Element = function(nsName, attrs, options) {
       }
     }
   }
+  for (var schemaKey in schemaAttrs) {
+    var schemaMatch = /^xmlns:?(.*)$/.exec(schemaKey);
+    if (schemaMatch && schemaMatch[1] && schemaMatch) {
```
Comment (herom) [inline lib/wsdl.js]: did you really want to check for `schemaMatch` twice here? 😄

File: lib/wsdl.js [hunk 2]
```diff
@@ -855,7 +862,9 @@ ElementElement.prototype.description = function(definitions, xmlns) {
   if (type) {
     type = splitQName(type);
     var typeName = type.name,
-      ns = xmlns && xmlns[type.prefix] || definitions.xmlns[type.prefix],
+      ns = xmlns && xmlns[type.prefix] ||
+        (definitions.xmlns[type.prefix] !== undefined && this.schemaXmlns[type.prefix]) ||
```
Comment (herom) [inline lib/wsdl.js]: when checking for `!== undefined` explicitly, we are aware that `null` would pass - is this intentional?

File: lib/wsdl.js (hunk 1 above)
Comment (PaoloSarti) [reply lib/wsdl.js]: fixed :)

File: lib/wsdl.js (hunk 2 above)
Comment (PaoloSarti) [reply lib/wsdl.js]: Anyway, I could change `definitions.xmlns[type.prefix] !== undefined` into `definitions.xmlns[type.prefix]`

Comment (herom) [reply lib/wsdl.js]: >  I checked the presence or absence of the property

good catch, I didn' t think of that... 👍

**Pull request merged**